src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -170,4 +170,25 @@ private String extractArgumentValue(String argsString,
         return value.trim();
     }
 
+    /**
+     * Represents a prefix's position in an arguments string.
+     */
+    public static class PrefixPosition {
+        private int startPosition;
+        private final Prefix prefix;
+
+        PrefixPosition(Prefix prefix, int startPosition) {
+            this.prefix = prefix;
+            this.startPosition = startPosition;
+        }
+
+        int getStartPosition() {
+            return startPosition;
+        }
+
+        Prefix getPrefix() {
+            return prefix;
+        }
+    }
+
```

Comment:
This was shifted into argument tokenizer from being in a separate class? Is it because only argument tokenizer is utilising this class?

Reply:
Yes, that is the reason.